main.go

```diff
@@ -253,11 +253,14 @@ func (cmd *InstallCmd) Run(ctx *Context) error {
 		err = runInModules([]string{d.Repository}, func(module string) error {
 
 			fmt.Printf("Checking module %s\n", module)
-			if err := os.Chdir(d.Path); err != nil {
-				return err
+			
+			if d.Path != "" {
+				if err := os.Chdir(d.Path); err != nil {
```

Comment:
Do you need to defer and change back to the previous working dir?

Reply:
That is what it's doing, but it's not really a problem that it stays in the previous daemon's path. No path is specified because the commands it needs to run can be done from any path. So if commands need to be run in a specific place, path will be specified, otherwise it can be left empty.